cmake/SearchForStuff.cmake

```diff
@@ -327,7 +336,14 @@ if (PKG_CONFIG_FOUND)
 
   #################################################
   # Find TBB
-  pkg_check_modules(TBB tbb)
+  find_package(TBB CONFIG REQUIRED)
```

Comment:
For backward compatibility, can we search via `find_package(TBB CONFIG)`  only if `    pkg_check_modules(TBB tbb)` fails?